core/openapi/schemas/party.yaml

```diff
@@ -1,103 +1,89 @@
-oneOf:
-  - type: object
+type: object
+description: |-
+  Identification of, and means of communication with, person responsible
+  for the resource.
+required:
+  - name
+properties:
+  identifier:
+    type: string
     description: |-
-      A reference to information about the responsible party.  For example,
-      a link to a VCard (see https://www.w3.org/TR/vcard-rdf/).
-    properties:
-      url:
-        type: string
-        format: uri
-        description: A URI providing a link to the contact.
-      roles:
-        $ref: roles.yaml
-    required:
-      - url
-  - type: object
+      A value uniquely identifying a party (individual or organization)
+  name:
+    type: string
+    description: |-
+      The name of the organization or the individual.
+  positionName:
+    type: string
+    description: Role or position of the responsible person.
+  organization:
+    type: string
+    description: Organization/affiliation of the individual/responsible person. In case of an organization, the name property should be used and this property is not to be used.
+  logo:
+    $ref: common/link.yaml
+    description: Graphic identifying a party
+  contactInfo:
```

Comment:
I think it would make sense to simplify this a little and move the fields defined in the contactInfo one level up. 
I'm aware that this is coming from ISO 19115, but it feels like a XML thing. It's a 0..1 relation, so there's no actual functionality behind it, e.g. to allow multiple of these objects. What do you think?

Reply:
Agreed.